hurumap_ke/templates/profile/profile_detail.html

```diff
@@ -989,5 +989,128 @@ <h1>Voter Registration 2017 </h1>
         </section>
       </div>
     </article>{% endif %}
+    {% if not gender.is_missing %}
+    <article class="clearfix">
+      <header class="section-contents">
+        <h1>Gender</h1>
+      </header>
+      <div class="section-container">
+        {% if not gender.hypertension_and_diabetes_sex.is_missing %}
+          <section class="clearfix stat-row">
+            <div class="column-full"
+                 id="chart-grouped_column-gender-hypertension_or_diabetes_sex"
+                 data-stat-type="scaled-percentage"
+                 data-chart-title="Occurence of Hypertension or Diabetes by sex"></div>
+            <div class="column-full"
+                 id="chart-grouped_column-gender-hypertension_or_diabetes_agegroup"
+                 data-stat-type="scaled-percentage" data-chart-title="Occurence of Hypertension and Diabetes by age group">
+            </div>
+          </section>{% endif %}
+
+        {% if not gender.cervical_cancer.is_missing %}
+          <section class="clearfix stat-row">
+            <div class="column-full"
+                 id="chart-grouped_column-gender-cervical_cancer"
+                 data-stat-type="scaled-percentage"
+                 data-chart-title="Women who have heard of Cervical Cancer">
+            </div>
+          </section>
+        {% endif %}
+        {% if not gender.prostate_cancer.is_missing %}
+          <section class="clearfix stat-row">
+            <div class="column-full"
+                 id="chart-grouped_column-gender-prostate_cancer"
+                 data-stat-type="scaled-percentage" data-chart-title="Men who have heard of Prostate Cancer">
+          </section>
+        {% endif %}
+
+
+        <section class="clearfix stat-row">
+          {% if not gender.physical_violence_perpetrator_sex.is_missing %}
```

Comment:
`if` statements must come _before_ `stat-row`, otherwise we end up with empty rows if data is missing.
![Screenshot from 2019-03-28 08-05-25](https://user-images.githubusercontent.com/1779590/55131601-4e924680-5130-11e9-94b8-db791bfa4589.png)